cmd/s2i/main.go

```diff
@@ -122,6 +122,10 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 				}
 			}
 
+			if cfg.ForcePull == true {
+				glog.Warning("DEPRECATED: The '--force-pull' option is deprecated. Use '--pull-policy' instead")
```

Comment:
Make all the deprecation info identical, see [here](https://github.com/openshift/source-to-image/blob/master/cmd/s2i/main.go#L163) and [here](https://github.com/openshift/source-to-image/blob/master/cmd/s2i/main.go#L167). Doing the latter two your way is also a good option, or making a function for that would be the best :wink: